### DISCUSSION TITLE
Nordic nRF5340 DK

### DISCUSSION THREAD

**Discussion by lboue** [Samsung/ioter#90] [General]
Hello,

Is it possible to also use a Nordic nRF5340 card too?
It seems certified and listed here: https://openthread.io/vendors/nordic-semiconductor#nrf5340

**Comment by sangsoolee** [comment]
@lboue 
Unfortunately, nRF5340 does not support RCP function yet.
The specifications of 5340 are too high-end to be used only as RCP.
I believe that Nordic think it will not be helpful to the customers who develop the product.
